lib/Prelude.dbl

```diff
@@ -87,6 +87,18 @@ pub method makeString {`re : {type X} -> Unit ->[|_] X, self : String}
     (extern dbl_strMake : Int -> String) n
   else `re ()
 
+pub method toList     = (extern dbl_strToListChr : String -> List Char) self
```

Comment:
Should this function be implemented as extern? I think it's expressible using other functions.

My concerns come from the principle to keep extern functions as simple as possible. With simpler set of extern functions it would be easier to add new backend for the interpreter/compiler.